### PR TITLE
Add 2.10.0 using Grayskull

### PR DESCRIPTION
**Pull request by rxm7706**
Removed Tests
Converted to Noarch
Cleanup to match graskull 

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,57 +1,43 @@
...
-    url: https://github.com/adamchainz/time-machine/archive/refs/tags/{{ version }}.tar.gz
-    sha256: 4c356560a2af1b3235da6deeeb68de67afda2bcfbe2a810e211ae4c5df3684a4
-
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/time_machine-{{ version }}.tar.gz
+  sha256: 64fd89678cf589fc5554c311417128b2782222dd65f703bf248ef41541761da0
 
 build:
-  skip: true  # [py<37]
-  script: cd dist && {{ PYTHON }} -m pip install . -vv
+  noarch: python
```
Comment (bollwyvl) [inline recipe/meta.yaml:13]: as this still uses [extension modules](https://github.com/adamchainz/time-machine/blob/2.10.0/setup.py#L16) it cannot be noarch.

Comment (rxm7706) [reply recipe/meta.yaml:13]: Thanks @bollwyvl I see, Let me try again to see if I can get this build working - Will start over with a new PR
